### DISCUSSION TITLE
Can I Upload a Document for Verification Instead of Taking a Photo?

### DISCUSSION THREAD

**Discussion by ceponatia** [community/community#36145] [GitHub Education] [answered]
Hi, I've tried three different browsers on two different devices and Github doesn't open up my camera on any of them, so I can't take a picture of my university document. Can't I upload a document instead? Having to print something from the university website and then take a picture of it is a bizarre process.

**Comment by FahdSeddik** [comment, marked as answer]
Hey @ceponatia 
I believe this is done for security/validation reasons. Check out this [link](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) for reference. Hope your problem gets solved!
